packages/aragon-api-react/src/index.js

```diff
@@ -15,6 +15,10 @@ const requestMenu = () => {
   postMessage('requestMenu', true)
 }
 
+const requestPath = (path, appId) => {
+  postMessage('requestPath', [path, appId])
```

Comment:
How come we use postMessage for this rather than the rpc abstraction provided by `@aragon/api`?